contracts/stellar-multicall/src/tests/test.rs

```diff
@@ -1,37 +1,45 @@
 #![cfg(test)]
 extern crate std;
 
-use soroban_sdk::testutils::Address as _;
+use soroban_sdk::testutils::{Address as _, BytesN as _};
 use soroban_sdk::{
-    contract, contractimpl, symbol_short, vec, Address, Env, IntoVal, Symbol, Val, Vec,
+    bytes, contract, contractimpl, vec, Address, Bytes, BytesN, Env, IntoVal, String, Val, Vec,
 };
+use soroban_token_sdk::metadata::TokenMetadata;
+use stellar_axelar_gas_service::event::GasPaidEvent;
+use stellar_axelar_gas_service::testutils::setup_gas_token;
+use stellar_axelar_gas_service::{AxelarGasService, AxelarGasServiceClient};
+use stellar_axelar_operators::{AxelarOperators, AxelarOperatorsClient};
```

Comment:
why are these being used? just created test contracts with minimal code needed to test the scenario to keep it simple